src/WebJobs.Script/ConsoleScriptJobHostEnvironment.cs

```diff
@@ -1,31 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using Microsoft.Extensions.Hosting;
```

Comment:
Does removing this have any impact in the context of a console app?

Reply:
I don't think so, as the IApplicationLifetime events will just be handled normally now. You can try it out though.